src/config/user.cr

```diff
@@ -5,21 +5,19 @@ struct DppmRestApi::Config::User
   API_KEY_SIZE = 63_u8
   include JSON::Serializable
   property api_key_hash : Scrypt::Password
-  setter groups : Set(Int32)
+  getter group_ids : Set(Int32)
+  property name : String
+  # :nodoc:
+  class_property all_groups : Array(Group) = Array(Group).new
```

Comment:
What's the point of having a second copy of this?

Reply:
That's not a copy, array is a class, so passed as a reference.
How do you access to all groups from an `User` instance then (without depending to `DppmRestApi`)?

Reply:
Oh, ok, you're right. Since it's just a reference it's no big deal. I still don't know how to do this when deserializing from JSON though...

Reply:
I've changed the approach by using a `GroupView`